lib/ansible/modules/windows/setup.ps1

```diff
@@ -318,11 +323,12 @@ if($gather_subset.Contains('platform')) {
 
     $ansible_facts += @{
         ansible_architecture = $win32_os.OSArchitecture
-        ansible_domain = $ip_props.DomainName
-        ansible_fqdn = ($ip_props.Hostname + "." + $ip_props.DomainName)
-        ansible_hostname = $env:COMPUTERNAME
+        ansible_domain = $domain_suffix
```

Comment:
We still have the same issue here, the value will be `.domain.com` when it should be `domain.com`.

Reply:
my brain is a pudding, sec